src/ResourceManager/Profile/Commands.Profile/Context/GetAzureRMContext.cs

```diff
@@ -26,6 +27,26 @@ namespace Microsoft.Azure.Commands.Profile
     [OutputType(typeof(PSAzureContext))]
     public class GetAzureRMContextCommand : AzureRMCmdlet
     {
+        /// <summary>
+        /// Gets the current default context.
+        /// </summary>
+        protected override AzureContext DefaultContext
```

Comment:
Looks good - add the test and this should be good - you should be able to test this with a unit test mocking out ICommandRuntime